sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -377,6 +377,21 @@ public void captureSession(final @NotNull Session session, final @Nullable Objec
     return sentryId;
   }
 
+  private List<Attachment> filterForTransaction(List<Attachment> attachments) {
```

Comment:
M: for nonprimitive types, lets always add `final @NotNull` or `final @Nullable`
a linter is on its way if we work together on https://github.com/getsentry/sentry-java/pull/1137/files#r548151954 :)